abft/check_against_db.go

```diff
@@ -22,35 +22,52 @@ import (
 	"github.com/0xsoniclabs/consensus/lachesis"
 )
 
-func CheckEpochAgainstDB(conn *sql.DB, epoch idx.Epoch) error {
+func SetupElection(conn *sql.DB, epoch idx.Epoch) (*CoreLachesis, *EventStore, map[hash.Event]*dbEvent, []*dbEvent, error) {
```

Comment:
Nit: I would consider having an auxiliary struct for this fivefold return, I feel that positional arguments are less clear when there are 5 of them. A matter of taste.